torch/utils/throughput_benchmark.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch._C
+
+class ThroughputBenchmark(object):
+    '''
+    This class is a wrapper around a c++ component throughput_benchmark::ThroughputBenchmark
+    responsible for executing a PyTorch module (nn.Module or ScriptModule)
+    under an inference server like load. It can emulate multiple calling threads
+    to a single module provided. In the future we plan to enhance this component
+    to support inter and intra-op parallelism as well as multiple models
+    running in a single process.
+
+    Please note that even though nn.Module is support, it might incur an overhead
+    from the need to hold GIL every time we execute Python code or pass around
+    inputs as Python objects. As soon as you have a ScriptModule version of your
+    model for inference deployment it is better to switch to using it in this
+    benchmark.
+
+    Example usage:
+
+        from torch.utils import ThroughputBenchmark
+
+        bench = ThroughputBenchmark(my_module)
+
+        for input in inputs:
+            # Both args and kwargs work, same as any PyTorch Module / ScriptModule
+            bench.add_input(input[0], x2=input[1])
```

Comment:
i'm not sure from reading the doc in this file on what `add_input` does. Does it add a set of inputs, and if you call `add_input` multiple times with different inputs it will pick one input set at random for a benchmark iteration?

Reply:
yes, it will pick up a random input at each iteration (almost, I pre-generate  random indexes). Will update this